streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -157,7 +157,7 @@ public void testStateOneThreadDeadButRebalanceFinish() throws InterruptedExcepti
 
         TestUtils.waitForCondition(
             () -> stateListener.numChanges == 2,
-            10 * 1000,
+            30_000,
```

Comment:
nit: make a constant for this value? helps to read and understand the time unit.